### PR TITLE
Snakify

### PR DESCRIPTION
**Pull request by abhishekr700**
Snakifying Device Tree

### CODE REVIEW

File: snake.mk [hunk 1]
```diff
@@ -1,9 +1,9 @@
 $(call inherit-product, device/asus/Z00L/full_Z00L.mk)
 
-# Inherit some common CM stuff.
-$(call inherit-product, vendor/cm/config/common_full_phone.mk)
+# Inherit some common Snake/AOSSP stuff.
+$(call inherit-product, vendor/snake/config/common_full_phone.mk)
 
-PRODUCT_NAME := cm_Z00L
+PRODUCT_NAME := snake_Z00L
```
Comment (vbajs) [inline snake.mk]: This should be `aossp_Z00L` not `snake_Z00L`

File: snake.mk [hunk 2]
```diff
@@ -1,9 +1,9 @@
 $(call inherit-product, device/asus/Z00L/full_Z00L.mk)
 
-# Inherit some common CM stuff.
-$(call inherit-product, vendor/cm/config/common_full_phone.mk)
+# Inherit some common Snake/AOSSP stuff.
```
Comment (vbajs) [inline snake.mk]: This could be `# Inherit some common stuff`
No need to say it ours even although it isn't

**Comment by vbajs** [comment]
No need for the branch `snakify`, so deleting..

**Comment by abhishekr700** [comment]
Okay, Locking this conversation now.

**Pull request merged**